{"uri":"file:///home/tarcisio/dev/phpfin/vendor/symfony/debug/Tests/MockExceptionHandler.php","root":{"kind":0,"name":"","children":[{"kind":512,"name":"Symfony\\Component\\Debug\\Tests","location":[11,0,11,40]},{"kind":1,"name":"ExceptionHandler","modifiers":4096,"location":[13,4,13,44],"associated":[{"kind":1,"name":"Symfony\\Component\\Debug\\ExceptionHandler"}]},{"kind":1,"name":"Symfony\\Component\\Debug\\Tests\\MockExceptionHandler","location":[15,0,23,1],"associated":[{"kind":1,"name":"Symfony\\Component\\Debug\\ExceptionHandler"}],"children":[{"kind":16,"name":"$e","modifiers":1,"location":[17,11,17,13],"scope":"Symfony\\Component\\Debug\\Tests\\MockExceptionHandler"},{"kind":32,"name":"handle","modifiers":1,"location":[19,4,22,5],"scope":"Symfony\\Component\\Debug\\Tests\\MockExceptionHandler","children":[{"kind":128,"name":"$e","location":[19,27,19,40],"scope":"handle","typeSource":1,"type":"Exception"}]}]}]}}